src/site/_includes/partials/post.njk

```diff
@@ -54,6 +54,32 @@
       </div>
     {% endif %}
 
+    {% if translation %}
+      {% if translation === true %}
+        <div class="w-banner w-banner--warning w-banner--body">
+          This page is not translated yet.
+          Consider improving it by contributing your translation
+          <a href="{{ page.inputPath | gitlocalizeLink(lang) }}">here</a>.
+        </div>
+      {% endif %}
+      {% if translation === 'machine' %}
+        <div class="w-banner w-banner--caution w-banner--body">
+          This page is automatically translated using Google Translation API.
```

Comment:
using _the_ Google Translation API

Reply:
Also, will this string itself need to be translated? 😆

Reply:
All template strings will need to be translated, eventually:)